pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["poetry>=0.12"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "flake8-pantsbuild"
+version = "0.1.0"
+description = "Various Flake8 lints used by the Pants project and its users."
+authors = ["Eric Arellano <ericarellano@me.com>"]
+license = "Apache-2.0"
+
+[tool.poetry.plugins."flake8.extension"]
+PNT = "flake8_pantsbuild:Plugin"
+
+[tool.poetry.dependencies]
+python = "~2.7 || ^3.5"
```

Comment:
Poetry supports the conventional `<`, `<=`, `==` etc. style of version strings, along with the NPM style used here. The `~` means support any patch version with this minor version and `^` means support any minor version with this major version. See https://python-poetry.org/docs/dependency-specification/#version-constraints.

Reply:
Feeling you have to explain this NPM style seems to tip the hat not to use the foreign style and instead stick to the conventions. Perhaps better would be to explain why you didn't want to stick to the conventions instead of explaining the syntax. My guess is you have a good reason.

Reply:
Fair, went with conventional version strings because my reason isn't good enough to justify the possible confusion. Thanks!